### ISSUE TITLE
Thank you for blazor!!! It rocks!!!

### ISSUE THREAD

**Issue by RealOrko** [dotnet/aspnetcore#18233]
Fellow programmers, been playing with blazor server side. I must say v3 is pretty good. Congradulations to the framework and tooling teams for getting this going. WebAssembly is the next wave and it will hopefully kill javascript with substitute languages like DOTNET Core and RUST.

I have been working as a full stack contractor in London doing horrible javascript front ends with awesome csharp back ends on linux.

You are going to be pressured hard to follow javascript frameworks, my answer to this is simply say "NO!".

Thank you for trying getting me out of this JavaScript hell of an ecosystem. :smile:

**Comment by RealOrko** [comment]
Looking forward to your Q1/2020 release of client side web assembly, There are so many WebForms/MVC/razor savvy devs in London working in finance, e-commerce, legal that are waiting for your gear to drop. 

We are done with javascript! Give us a strongly typed language!

Is this the end of typescript?

**Comment by RealOrko** [comment]
@ahejlsberg

You worked under https://en.wikipedia.org/wiki/Niklaus_Wirth

You are my hero. Been folowing you for ages. Thoughts?

**Comment by rynowak** [comment]
> We are done with javascript! Give us a strongly typed language!
> 
> Is this the end of typescript?

Firstly, @RealOrko thanks for everything positive you have to say about Blazor, we're happy to hear that you love it!

Secondly, it's definitely **not** our goal to stop people from using JavaScript or TypeScript if that's what they like. Blazor is another option among a whole bunch of great choices. I think what speaks to a lot of folks is that they like .NET and C# and want to use it for everything they do - now you can! I imagine a lot of JavaScript developers (or developers using any toolset) feel as happy and productive as you do using .Net.
